index.html

```diff
@@ -132,120 +170,135 @@ <h2 class="card-title">Tonic</h2>
+                <div class="skills-arrow">
+                  <h3>Skills</h3>
+                  <img src="./images/arrow2.png" alt="Arrow right">
+                </div>
+                <div class="line"></div>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
     </section>
 
     <footer class="con-form-wrapper">
-      <div class="contact-form">
-        <h2 class="contact-me">Contact me</h2>
-        <p class="contact-description">If you have an application you are interested in developing, a feature that you
-          need built or a project that needs coding. I’d love to help with it.</p>
-        <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mrgjgzbv"
-          method="post">
-          <div class="frm-inputs-cont">
-            <label for="full-name"></label>
-            <input class="con-frm-input" type="text" name="name" id="full-name" placeholder="First and Last Name"
-              required maxlength="30">
-            <label for="email-address"></label>
-            <input class="con-frm-input" type="email" name="_replyto" id="email-address"
-              placeholder="email@microverse.org" required>
-            <label for="message"></label>
-            <textarea class="con-frm-mess" rows="5" name="message" id="message" placeholder="Write your message here"
-              required maxlength="500"></textarea>
-            <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
-            <input class="frm-button" type="submit" value="Get In Touch">
-          </div>
-        </form>
+      <div class="footer-connect-container">
+        <div class="contact-form">
+          <h2 class="contact-me">Contact me</h2>
+          <p class="contact-description">If you have an application you are interested in developing, a feature that you
+            need built or a project that needs coding. I’d love to help with it.</p>
+          <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mrgjgzbv"
+            method="post">
+            <div class="frm-inputs-cont">
+              <label for="full-name"></label>
+              <input class="con-frm-input" type="text" name="name" id="full-name" placeholder="First and Last Name"
+                required maxlength="30">
+              <label for="email-address"></label>
+              <input class="con-frm-input" type="email" name="_replyto" id="email-address"
+                placeholder="email@microverse.org" required>
+              <label for="message"></label>
+              <textarea class="con-frm-mess" rows="5" name="message" id="message" placeholder="Write your message here"
+                required maxlength="500"></textarea>
+              <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+              <input class="frm-button" type="submit" value="Get In Touch">
```

Comment:
### Good job :clap: :clap: :+1:
 Your project is perfectly well designed and you really followed the same template's design however one of the requirements says that when designing your desktop version the mobile version should not be broken. 
- [ ] KIndly your design would look better and sticks to The template by adding some `padding-top` on in the `textarea` in the form; check the image bellow :point_down:  and perform yours.
- [ ] I would suggest aligning very well the `get in touch` button still in the form. check the image bellow :point_down: 

Your design | The template
---------------- | -----------------
![Screenshot from 2021-12-11 15-31-59](https://user-images.githubusercontent.com/65068771/145678967-a40bc5a6-b847-462e-a0f7-296437cbf749.png) | ![Screenshot from 2021-12-11 15-30-48](https://user-images.githubusercontent.com/65068771/145678970-11cc7754-a222-47c4-a190-17e07c328b10.png)